.buildkite/pipeline.yml

```diff
@@ -1,20 +1,54 @@
 agents:
-  queue: opensource-arm-mac-cocoa-12
+  queue: macos-12-arm
 
 steps:
 
-  - label: ":test_tube:"
+  - label: ":test_tube: 2.8.1"
     timeout_in_minutes: 10
+    env:
+      FLUTTER_DIR: "/opt/flutter/2.8.1/bin/flutter"
```

Comment:
Almost a nit, but could be confusing that FLUTTER "DIR" is actually the executable.